<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="11" column="1">
    <widget class="QTextBrowser" name="textReceived"/>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="syncBtn">
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="receivedLabel">
     <property name="text">
      <string>Received</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="updateFilesBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="connectBtn">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="echoBtn">
     <property name="text">
      <string>Echo</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="echoLineEdit"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="dirLineEdit">
     <property name="text">
      <string>/home/timin/WorkSpace/qtFileClient/</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="ipLineEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portLineEdit">
       <property name="text">
        <string>6060</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
